src/blocks/cpu.rs

```diff
@@ -46,12 +49,16 @@ pub struct CpuConfig {
     #[serde(default = "CpuConfig::default_critical")]
     pub critical: u64,
 
-    /// Display frequency
-    #[serde(default = "CpuConfig::default_frequency")]
-    pub frequency: bool,
```

Comment:
OK, so we don't actually want to remove this, as it will break existing user configurations. Instead, we should permit this key and set `format` to `{utilization} {frequency}GHz` if it is present but `format` is absent. This is a pattern we've used elsewhere when introducing a `format` key.

Reply:
added frequency back and its documentation back